src/poll.rs

```diff
@@ -1161,7 +1161,7 @@ impl Poll {
         self.readiness_queue.poll(&mut events.inner);
 
         // Return number of polled events
-        Ok(events.len())
+        Ok(events.iter().count())
```

Comment:
Should we not return the number of polled events since we do not know the exact number?

Reply:
Yeah, I think some return value like that could be good, but it will have to be punted until 0.7. We can open an issue to track that change. The ret value today will have to stay the same :(